### ISSUE TITLE
Save Button not always available

### ISSUE THREAD

**Issue by tmginzburg** [RallyCommunity/query-counter#13]
I am using the app on a dashboard page. Sometimes when going to Edit App Settings, the settings are displayed but only part of the lower section is shown and the Save / Cancel buttons are not available.  There is no option to scroll down, so I have to close the app and go back in, losing any changes I made if I didn't notice the missing Save button.

**Comment by gmonahan** [comment]
Recommend you get the latest code as I believe this was fixed in the last build. The work-around I used before was to click in the freeform text area and then use the tab key to force the bottom of the form to render.

**Comment by krmorse** [comment]
Yep, should be fixed in master, fix just has not been promoted into the community app catalog in-product yet.  Maybe @anagram15 can speak to timeline there?

**Comment by gmonahan** [comment]
@tmginzburg - did you want to Close this if you've pulled the latest build?
